xds/internal/xdsclient/transport/transport.go

```diff
@@ -57,17 +57,18 @@ type adsStream = v3adsgrpc.AggregatedDiscoveryService_StreamAggregatedResourcesC
 // protocol version.
 type Transport struct {
 	// These fields are initialized at creation time and are read-only afterwards.
-	cc                  *grpc.ClientConn        // ClientConn to the mangement server.
-	serverURI           string                  // URI of the management server.
-	updateHandler       UpdateHandlerFunc       // Resource update handler. xDS data model layer.
-	adsStreamErrHandler func(error)             // To report underlying stream errors.
-	lrsStore            *load.Store             // Store returned to user for pushing loads.
-	backoff             func(int) time.Duration // Backoff after stream failures.
-	nodeProto           *v3corepb.Node          // Identifies the gRPC application.
-	logger              *grpclog.PrefixLogger   // Prefix logger for transport logs.
-	adsRunnerCancel     context.CancelFunc      // CancelFunc for the ADS goroutine.
-	adsRunnerDoneCh     chan struct{}           // To notify exit of ADS goroutine.
-	lrsRunnerDoneCh     chan struct{}           // To notify exit of LRS goroutine.
+	cc                  *grpc.ClientConn         // ClientConn to the mangement server.
+	serverURI           string                   // URI of the management server.
+	updateHandler       UpdateHandlerFunc        // Resource update handler. xDS data model layer.
+	adsStreamErrHandler func(error)              // To report underlying stream errors.
+	reportSendReq       func(*UpdateChannelInfo) // To report underlying stream successful sends.
```

Comment:
These can also be renamed as `onUpdate`, `onStreamError/onConnectionError`, and `onSend/onResourceSend`.